pkg/validation/app.go

```diff
@@ -174,6 +181,18 @@ func (v *Validator) validateName(ctx context.Context, cr v1alpha1.App) error {
 	return nil
 }
 
+func (v *Validator) validateTargetNamespace(ctx context.Context, cr v1alpha1.App) error {
+	if key.InCluster(cr) {
+		if strings.HasPrefix(cr.Namespace, "org-") {
```

Comment:
I think a brief comment here for why we need this would be good.

Reply:
Will do, thanks!